platform/ios/src/MGLMapView.mm

```diff
@@ -1230,11 +1230,6 @@ - (void)validateDisplayLink
     BOOL isVisible = self.superview && self.window;
     if (isVisible && ! _displayLink)
     {
-        if (_mbglMap && self.mbglMap.getConstrainMode() == mbgl::ConstrainMode::None)
-        {
-            self.mbglMap.setConstrainMode(mbgl::ConstrainMode::HeightOnly);
-        }
-
```

Comment:
I think removing this will cause the issue fixed in #3887 to regress.

Reply:
@1ec5 is that still an issue? IMO If yes, I think that we could initialize the Map object with a fake size of e.g. 64x64 and that should be enough, no?

Reply:
/cc @julianrex

Reply:
> I think removing this will cause the issue fixed in #3887 to regress.

Looks like it does.

Reply:
Per #3887, we don’t have control over the order in which the camera parameters are individually applied, nor over when they’re applied. If we did have control over that, I agree that a “pending coordinate” pattern would be more appropriate.

> On iOS, the only guarantee we have timing-wise is that all the inspectables are applied after initialization but before the view is added to a window.

Reply:
Per #3887, we don’t have control over the order in which the camera parameters are individually applied, nor over when they’re applied. If we did have control over that, I agree that a “pending coordinate” pattern would be more appropriate.

> On iOS, the only guarantee we have timing-wise is that all the inspectables are applied after initialization but before the view is added to a window.